python/py-me-al/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        dabeaz me-al 0f2e04aecff88b73a25fe69e789f3ade26683866
+name                py-${github.project}
+version             2018.11.24
+revision            0
+categories-append   lang
+license             none
```

Comment:
I don't think this is a valid value... one could/should ask upstream what their intended license is

Reply:
This is a valid and correct value. Please see https://trac.macports.org/wiki/PortfileRecipes#license